### PR TITLE
reminders

### PR DESCRIPTION
**Pull request by g-vega-cl**
purpose:
 - Send reminders 2h and 24h before an appointment

Trello:
- https://trello.com/c/JqqhnjmB/70-appointments-client-should-receive-a-reminder-text-message-24-hrs-before-an-upcoming-appointment
- https://trello.com/c/dEeCZ0KL/71-appointments-client-should-receive-a-reminder-text-message-2-hrs-before-an-upcoming-appointment

### CODE REVIEW

**Comment by bkkilaru** [comment]
Per call, let's move to additional cron job so all automated messages are managed in a common way

File: src/background_jobs/cronSchedules.ts [hunk 1]
```diff
@@ -14,6 +18,12 @@ const runCronSchedules = () => {
     scheduled: true,
     timezone: 'America/Los_Angeles',
   });
+
+  // Loop appointment reminders every 10 minutes.
+  cron.schedule('*/10 * * * *', scheduleAppointmentReminders, {
```
Comment (bkkilaru) [inline src/background_jobs/cronSchedules.ts:23]: This is every 10th minute instead of every 10 minutes? How often did you want to check?

Comment (g-vega-cl) [reply src/background_jobs/cronSchedules.ts:23]: I think that's every 10 minutes. node-cron is weird.
Source:
https://www.npmjs.com/package/node-cron
And it says that to run every 2 minutes we should:
`var cron = require('node-cron');

cron.schedule('*/2 * * * *', () => {
  console.log('running a task every two minutes');
});`
And I checked it in my PC, and it does run every 2 min, so If we change 2 to 10 it should work every 10 min. But I'm double checking anyways.
Screen Shot 2021-08-12 at 9 00 46

Comment (g-vega-cl) [reply src/background_jobs/cronSchedules.ts:23]: Tested with 10 min:
Screen Shot 2021-08-12 at 9 39 09

**Review by bkkilaru — APPROVED** [review]

**Pull request merged**